calicoctl/resourcemgr/networkpolicy.go

```diff
@@ -39,14 +43,35 @@ func init() {
 		},
 		func(ctx context.Context, client client.Interface, resource ResourceObject) (ResourceObject, error) {
 			r := resource.(*api.NetworkPolicy)
+			if strings.HasPrefix(r.Name, conversion.K8sNetworkPolicyNamePrefix) {
+				return nil, cerrors.ErrorOperationNotSupported{
+					Operation: "create or apply",
+					Identifier: fmt.Sprintf("GlobalNetworkPolicy(%s/%s)"),
+					Reason: "NetworkPolicy resources with 'knp.' prefixed names cannot be modified using calicoctl",
```

Comment:
A better reason might be "Kubernetes network policies cannot be managed using calicoctl" or something.

Maybe "Kubernetes network policies must be managed through the kubernetes API"?

Reply:
In either case, "modified" doesn't make sense in the "create" function.